tools/cudaq-qpud/nvqc_proxy.py

```diff
@@ -256,7 +262,8 @@ def do_POST(self):
         NUM_GPUS = 0
     MPI_FOUND = (shutil.which('mpiexec') != None)
     SUDO_FOUND = (shutil.which('sudo') != None)
-    WATCHDOG_TIMEOUT_SEC = int(os.environ.get('WATCHDOG_TIMEOUT_SEC', 0))
+    WATCHDOG_TIMEOUT_SEC = int(
+        os.environ.get('WATCHDOG_TIMEOUT_SEC', WATCHDOG_TIMEOUT_SEC))
```

Comment:
I'm curious if this change was required or not. Wasn't WATCHDOG_TIMEOUT_SEC initialized to 0 anyway? If so, it seems like it would be functionally the same.